transip/service/dns.py

```diff
@@ -0,0 +1,50 @@
+"""
+DNS related objects
```

Comment:
Disabling R0903 is alright, but I don't think you want to disable C0111 globally.
As de docs say: `Missing %s docstring Used when a module, function, class or method has no docstring. Some special methods like __init__ doesn’t necessary require a docstring.`
So the check would be disabled not only for modules, but also for classes and methods.
It would be better to just disable it for one module, but I don't think that's possible.
So, long story short: I think it's better to include a short generic docstring than to disable the check for them globally.